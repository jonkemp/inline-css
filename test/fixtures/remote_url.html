<html>
  <head>
    <link rel="stylesheet" href="https://unpkg.com/normalize.css@8.0.1/normalize.css">
  </head>
  <body>
    <h1>hello</h1></body></html>
