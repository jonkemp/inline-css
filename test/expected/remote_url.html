<html style="-webkit-text-size-adjust: 100%; line-height: 1.15;">
  <head>
    
  </head>
  <body style="margin: 0;">
    <h1 style="font-size: 2em; margin: 0.67em 0;">hello</h1>
  </body>
</html>
